README.md

```diff
@@ -1,56 +1,32 @@
-![Terraform Provider Iterative](https://static.iterative.ai/img/cml/banner-terraform.png)
+![TPI](https://static.iterative.ai/img/cml/banner-terraform.png)
 
-# Iterative Provider [![](https://img.shields.io/badge/-documentation-5c4ee5?logo=terraform)](https://registry.terraform.io/providers/iterative/iterative/latest/docs)
+# Terraform Provider Iterative (TPI)
 
-The Iterative Provider is a Terraform plugin that enables full lifecycle
-management of computing resources for machine learning pipelines, including GPUs, from your favorite cloud vendors.
+[![docs](https://img.shields.io/badge/-docs-5c4ee5?logo=terraform)](https://registry.terraform.io/providers/iterative/iterative/latest/docs)
+[![tests](https://img.shields.io/github/workflow/status/iterative/terraform-provider-iterative/Test?label=tests&logo=GitHub)](https://github.com/iterative/terraform-provider-iterative/actions/workflows/test.yml)
+[![Apache-2.0](https://img.shields.io/badge/licence-Apache%202.0-blue)](https://github.com/iterative/terraform-provider-iterative/blob/master/LICENSE)
 
-The Iterative Provider makes it easy to:
+- **Provision Resources**: create cloud compute & storage resources without reading pages of documentation
```

Comment:
Should we mention GPUs? It is good at least for SEO.

Reply:
💯

Reply:
already mentioned in the one-liner above. Probably odd to repeat?

Reply:
Even if it's odd to repeat, _compute_ might be a bit too general.

Reply:
Right, **compute** is too abstract.

**GPU** term is used once in the entire page. We definitely need to repeat it. The question is where :) 

I'd add some comment in tutorial for example on how to change the machine to GPU with a comment that GPU should be enabled for you.

Reply:
```suggestion
- **Provision Resources**: create cloud compute (CPU, GPU, RAM) & storage resources without reading pages of documentation
```